app/controllers/api_secrets_controller.rb

```diff
@@ -23,7 +23,7 @@ def destroy
     if @secret.destroy
       flash[:notice] = "Your API Key has been revoked."
     else
-      flash[:error] = "An error occurred. Please try again or send an email to: #{SiteConfig.email_addresses[:default]}"
+      flash[:error] = "An error occurred. Please try again or send an email to: #{SiteConfig.email_addresses[:contact]}"
```

Comment:
You'll notice this pattern. Everywhere we tell people to contact us, we should direct them to the `:contact` email